go/vt/sqlparser/ast.go

```diff
@@ -1570,6 +1583,102 @@ func (node *ValuesFuncExpr) WalkSubtree(visit Visit) error {
 	)
 }
 
+// ConvertExpr represents a call to CONVERT(expr, type)
+// CONVERT(expr USING transcoding_name) it not supported
+type ConvertExpr struct {
+	Expr Expr
+	Type *ConvertType
+}
+
+// Format formats the node.
+func (node *ConvertExpr) Format(buf *TrackedBuffer) {
+	buf.Myprintf("convert(%v, %v)", node.Expr, node.Type)
+}
+
+// WalkSubtree walks the nodes of the subtree.
+func (node *ConvertExpr) WalkSubtree(visit Visit) error {
+	if node == nil {
+		return nil
+	}
+	return Walk(
+		visit,
+		node.Expr,
+		node.Type,
+	)
+}
+
+// ConvertType represents the type in call to CONVERT(expr, type)
+type ConvertType struct {
+	Type    string
+	Length  *SQLVal
+	Scale   *SQLVal
+	Charset string
+}
+
+// ConvertType.Type
+const (
+	Binary   = "binary"
+	Char     = "char"
+	NChar    = "nchar"
+	Decimal  = "decimal"
+	Signed   = "signed"
+	Unsigned = "unsigned"
+	Date     = "date"
+	Datetime = "datetime"
+	Time     = "time"
+	Json     = "json"
```

Comment:
golint tells me this var name should be `JSON`

go/vt/sqlparser/ast.go:1591:2: const Json should be JSON